velox/functions/remote/server/RemoteFunctionService.cpp

```diff
@@ -102,11 +98,35 @@ void RemoteFunctionServiceHandler::invokeFunction(
           outputType,
           getFunctionName(functionPrefix_, functionHandle.get_name())),
       &execCtx};
+
   exec::EvalCtx evalCtx(&execCtx, &exprSet, inputVector.get());
+  if (!request->get_throwOnError()) {
+    *evalCtx.mutableThrowOnError() = false;
+  }
 
   std::vector<VectorPtr> expressionResult;
   exprSet.eval(rows, evalCtx, expressionResult);
 
+  auto evalErrors = evalCtx.errors();
+  if (evalErrors != nullptr && evalErrors->hasError()) {
```

Comment:
we chatted offline, but it would be nice to find a more efficient way to serialize these strings. Maybe writing them to a StringView vector then just serializing it?